gravitee-management-api-service/src/main/java/io/gravitee/management/service/ApiService.java

```diff
@@ -60,7 +60,7 @@
     
     ApiEntity rollback(String apiId, UpdateApiEntity api);
 
-    String exportAsJson(String apiId, MembershipType membershipType);
+    String exportAsJson(String apiId, MembershipType membershipType, String... filteredFields);
```

Comment:
Because ApiEntity need to know your membership to export some datas
see https://github.com/gravitee-io/gravitee-management-rest-api/blob/master/gravitee-management-api-model/src/main/java/io/gravitee/management/model/ApiEntity.java#L45